docs/contributing.md

```diff
@@ -1,5 +1,5 @@
 # Contributing to `pmdsky-debug`
-Contributions are welcome, no matter how small! This can be new symbols, new type definitions, changes to `resymgen`, bug fixes, documentation, etc. Make sure add yourself to [`CREDITS.txt`](../CREDITS.txt) if you're a new contributor!
+Contributions are welcome, no matter how small! This can be new symbols, new type definitions, changes to `resymgen`, bug fixes, documentation, etc. Make sure to add yourself to [`CREDITS.txt`](../CREDITS.txt) if you're a new contributor!
```

Comment:
For the commit history, separate PR please

Reply:
https://github.com/UsernameFodder/pmdsky-debug/pull/213